passes/cmds/stat.cc

```diff
@@ -206,6 +209,89 @@ struct statdata_t {
 			}
 
 			if (!cell_area.empty()) {
+				// check if cell_area provides a area calculator
+				if (cell_area.count(cell->type)) {
+					cell_area_t cell_data = cell_area.at(cell->type);
+					if (cell_data.single_parameter_area.size() > 0) {
+						// assume that we just take the max of the A,B,Y ports
+
+						int width_a = cell->hasPort(ID::A) ? GetSize(cell->getPort(ID::A)) : 0;
+						int width_b = cell->hasPort(ID::B) ? GetSize(cell->getPort(ID::B)) : 0;
+						int width_y = cell->hasPort(ID::Y) ? GetSize(cell->getPort(ID::Y)) : 0;
+						int width_q = cell->hasPort(ID::Q) ? GetSize(cell->getPort(ID::Q)) : 0;
+						int max_width = max<int>({width_a, width_b, width_y, width_q});
+						if (!cell_area.count(cell_type)) {
+							cell_area[cell_type] = cell_data;
+						}
+						if (cell_data.single_parameter_area.size() > max_width - 1u) {
+							cell_area.at(cell_type).area = cell_data.single_parameter_area.at(max_width - 1);
+							cell_area.at(cell_type).is_sequential = cell_data.is_sequential;
+
+						} else {
+							printf("too small single_parameter_area %s %d %f\n", cell_type.c_str(), max_width,
+							       cell_data.single_parameter_area.back());
```

Comment:
Why are you using `printf()` instead of `log()`?  Are these intended for the user or were they for debugging and need removing?